code/modules/antagonists/blob/blob/blobs/shield.dm

```diff
@@ -20,14 +21,37 @@
 
 /obj/structure/blob/shield/update_icon()
 	..()
-	if(obj_integrity <= 75)
-		icon_state = "blob_shield_damaged"
-		name = "weakened strong blob"
-		desc = "A wall of twitching tendrils."
+	if(obj_integrity < max_integrity * 0.5)
+		icon_state = "[initial(icon_state)]_damaged"
+		name = "weakened [initial(name)]"
+		desc = "[damaged_desc]"
 		atmosblock = FALSE
 	else
 		icon_state = initial(icon_state)
 		name = initial(name)
 		desc = initial(desc)
 		atmosblock = TRUE
 	air_update_turf(1)
+
+/obj/structure/blob/shield/reflective
+	name = "reflective blob"
+	desc = "A solid wall of slightly twitching tendrils with a reflective glow."
+	damaged_desc = "A wall of twitching tendrils with a reflective glow."
+	icon_state = "blob_glow"

```

Comment:
The damaged state for this sprite is different from shield blob's damaged state; here is a free pair of sprites. http://puu.sh/BWf7z/8c78d2aabe.dmi